### PR TITLE
finish the login flow perks

### PR DESCRIPTION
**Pull request by erikgalajda**
- add guard
- remove status component
- redirect from login to home if already logged in

### CODE REVIEW

File: apps/todoapp/src/app/todos/services/auth.service.ts [hunk 1]
```diff
@@ -21,12 +23,12 @@ export class AuthService {
           // User is authenticated, proceed with your application
           console.log('User is authenticated:', user);
           this.user$.next(user as DiscordUser);
-        },
-        (error) => {
-          // 403 response, user is not authenticated
```
Comment (danielbubanec) [inline apps/todoapp/src/app/todos/services/auth.service.ts:26]: Unauthenticated response code should be 401 not 403. 403 is for unauthorized access with correct token but forbidden access to some resource.

File: apps/todoapp/src/app/todos/components/todos/todos.component.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export class TodosComponent {
   constructor(private authService: AuthService) {}
 
   ngOnInit() {
-    this.authService.checkAuthStatus();
+    // this.authService.checkAuthStatus();
```
Comment (danielbubanec) [inline apps/todoapp/src/app/todos/components/todos/todos.component.ts]: Remove commented code.

File: apps/todoapp/src/app/todos/services/auth.service.ts [hunk 3]
```diff
@@ -21,12 +23,12 @@ export class AuthService {
...
-          console.error('User is not authenticated:', error);
-          this.router.navigate(['/login']);
         }
+        // (error) => {
```
Comment (danielbubanec) [inline apps/todoapp/src/app/todos/services/auth.service.ts]: Remove commented code.

**Review by danielbubanec — CHANGES_REQUESTED** [review]

File: apps/todoapp/src/app/todos/services/auth.service.ts (hunk 3 above)
Comment (erikgalajda) [reply apps/todoapp/src/app/todos/services/auth.service.ts]: we need that for now but maybe we can get rid of it in the future when we make the authguard to take care of. I will explain the problem in our next sync. :)

**Pull request merged**